utils/node_local_test.py

```diff
@@ -1999,8 +2009,44 @@ def _helper(obj):
                 if obj.dfuse.stop():
                     obj.fatal_errors = True
             return rc
+
+        needs_dfuse_with_opt.record_wrap(method.__name__, self.caching_variants)
         return _helper
 
+    @staticmethod
+    def get_excluded_versions(name):
```

Comment:
> missing-function-docstring, Missing function or method docstring

The pylint needs to be resolved before merging, and it will make it easier to understand the interface of these functions for reviewing

Reply:
Fixed.